Cloud Application Manager/Core Concepts/lifecycle-editor.md

```diff
@@ -46,6 +46,18 @@ If you make event script changes in the event tab and try to close editor (witho
 
 ![Save the changes when close Editor](../../images/cloud-application-manager/lifecycle-editor-3-1.png)
 
+**Edit Event Timeout**
+
+Events allow operator to setup a maximum execution timeout in order to avoid long-consuming executions. Execution timeout can be defined in [Script boxes](../Automating Deployments/script-box.md) and modified at instance level.
```

Comment:
```suggestion
You can set up a maximum execution timeout in an event script in order to avoid long-consuming executions. Execution timeout can be defined in [Script boxes](../Automating Deployments/script-box.md) and modified at instance level.
```
Is it possible to just set it at the instance level, even if it has not been defined at box level? I guess so, you should update wording to reflect it